### DISCUSSION TITLE
useDeno content doesn't update

### DISCUSSION THREAD

**Discussion by lcswillems** [alephjs/aleph.js#189] [Q&A]
For example, if I put this in a component:

```
const date = useDeno(async () => {
  const res = await fetch(
    "http://worldtimeapi.org/api/timezone/Europe/London.txt"
  );
  return res.text();
});
```

The date is always the same when I refresh the page. Deno seems to cache the value but this is unexpected behavior for server-side rendering.

**Comment by hazae41** [comment]
The factory function is only executed on initial build, not on every refresh

You can pass "true" to the useDeno hook to fetch client-side on each refresh

https://alephjs.org/docs/advanced-features/use-deno-hook

**Comment by lcswillems** [comment]
Okay thank you!
